test/e2e/storage/vsphere_stress.go

```diff
@@ -0,0 +1,180 @@
+		// Resulting 120 Volumes and POD Creation. Volumes will be provisioned with each different types of Storage Class,
+		// Each iteration creates PVC, verify PV is provisioned, then creates a pod, verify volume is attached to the node, and then delete the pod and delete pvc.
+
+		Expect(os.Getenv("VCP_STRESS_INSTANCES")).NotTo(BeEmpty(), "ENV VCP_STRESS_INSTANCES is not set")
+		instances, err = strconv.Atoi(os.Getenv("VCP_STRESS_INSTANCES"))
+		Expect(err).NotTo(HaveOccurred(), "Error Parsing VCP-STRESS-INSTANCES")
+		Expect(instances <= volumesPerNode*len(nodeList.Items)).To(BeTrue(), fmt.Sprintf("Number of Instances should be less or equal: %v", volumesPerNode*len(nodeList.Items)))
+		Expect(instances > 3).To(BeTrue(), "VCP_STRESS_INSTANCES should be greater than 3 to utilize all 4 types of storage classes")
+
+		Expect(os.Getenv("VCP_STRESS_ITERATIONS")).NotTo(BeEmpty(), "ENV VCP_STRESS_ITERATIONS is not set")
+		iterations, err = strconv.Atoi(os.Getenv("VCP_STRESS_ITERATIONS"))
+		Expect(err).NotTo(HaveOccurred(), "Error Parsing VCP_STRESS_ITERATIONS")
+		Expect(iterations > 0).To(BeTrue(), "VCP_STRESS_ITERATIONS should be greater than 0")
+
+		Expect(os.Getenv("VSPHERE_SPBM_POLICY_NAME")).NotTo(BeEmpty(), "ENV VSPHERE_SPBM_POLICY_NAME is not set")
+		Expect(os.Getenv("VSPHERE_DATASTORE")).NotTo(BeEmpty(), "ENV VSPHERE_DATASTORE is not set")
+	})
+
+	It("vsphere stress tests", func() {
+		scArrays := make([]*storageV1.StorageClass, 4)
+		// Create default vSphere Storage Class
+		By(fmt.Sprintf("Creating Storage Class : %v", storageclass1))
+		scDefault, err := client.StorageV1().StorageClasses().Create(getVSphereStorageClassSpec(storageclass1, nil))
+		Expect(err).NotTo(HaveOccurred())
+		defer client.StorageV1().StorageClasses().Delete(storageclass1, nil)
+		scArrays[0] = scDefault
+
+		// Create Storage Class with vSan Parameters
+		By(fmt.Sprintf("Creating Storage Class : %v", storageclass2))
+		var scVSanParameters map[string]string
+		scVSanParameters = make(map[string]string)
+		scVSanParameters[Policy_HostFailuresToTolerate] = "1"
+		scVSan, err := client.StorageV1().StorageClasses().Create(getVSphereStorageClassSpec(storageclass2, scVSanParameters))
+		Expect(err).NotTo(HaveOccurred())
+		defer client.StorageV1().StorageClasses().Delete(storageclass2, nil)
+		scArrays[1] = scVSan
+
+		// Create Storage Class with SPBM Policy
+		By(fmt.Sprintf("Creating Storage Class : %v", storageclass3))
+		var scSPBMPolicyParameters map[string]string
+		scSPBMPolicyParameters = make(map[string]string)
+		spbmpolicy := os.Getenv("VSPHERE_SPBM_POLICY_NAME")
+		Expect(spbmpolicy).NotTo(BeEmpty())
+		scSPBMPolicyParameters[SpbmStoragePolicy] = spbmpolicy
+		scSPBMPolicy, err := client.StorageV1().StorageClasses().Create(getVSphereStorageClassSpec(storageclass3, scSPBMPolicyParameters))
+		Expect(err).NotTo(HaveOccurred())
+		defer client.StorageV1().StorageClasses().Delete(storageclass3, nil)
+		scArrays[2] = scSPBMPolicy
+
```

Comment:
can the 4 storage classes above be created in a `for` loop? Eg the classname constants can be elements in an array which can be iterated over to create the 4 storageClasses. Yes, you'd need to still do the special setup for each storageClass outside of the loop, but this would still simplify the code some.

Reply:
created 4 stotage classes in the for loop.